Sources/ArcGISToolkit/Components/FeatureFormView/FormViewModel.swift

```diff
@@ -31,6 +31,12 @@ import SwiftUI
     /// The list of visible form elements.
     @Published var visibleElements = [FormElement]()
     
+    /// A Boolean value indicating whether the full screen text input is presented.
+    @Published var fullScreenTextInputIsPresented = false
+        
+    /// Text that will be displayed in a view showing multi line text.
+    @Published var fullScreenText = ""
```

Comment:
This property is never written to. Currently, when the sheet is brought up, the element's current value is not injected.

<p align="center">
  <img width="50%" src="https://github.com/user-attachments/assets/1f2a340c-570c-4b23-8cfc-ee22853a57c4">
</p>